.github/workflows/company4-e2e.yml

```diff
@@ -0,0 +1,105 @@
+name: E2E Testing
+
+on:
+  push: 
+    branches: [main]
+  repository_dispatch:
+    types: 
+      - company4-algorithm-1
```

Comment:
We configured our events to be distinct from yours to have a completely independent trigger/flow

Reply:
Looks good here